sql/moz-fx-data-shared-prod/firefox_ios_derived/funnel_retention_clients_week_2_v1/checks.sql

```diff
@@ -1,5 +1,8 @@
-#warn
-{{ is_unique(["client_id"]) }}
+{#
+-- Disabled for now due to known duplication issue in Fenix data, see: bug-1887708
```

Comment:
This is the metrics ping size bug https://bugzilla.mozilla.org/show_bug.cgi?id=1887708
I would recommend putting the full link just so it's easier to go to

Reply:
yeah, I'm never sure if we should be including full links or not to avoid people from outside Mozilla attempting to access our resources.